cpplang/tree.py

```diff
@@ -618,6 +618,8 @@ class ParmVarDecl(Declaration):
 class FieldDecl(Declaration):
     attrs = ("type", "name", "init", "bitwidth", "attributes", "type_qualifier",)
 
+class CXXDefaultInitExpr(Declaration):
+    attrs = ("expression")
```

Comment:
You should add a `,` here so that  `attrs` is considered a tuple. I'm surprised this works o_O

Reply:
Thanks! I didnt know/remember that.
I think it's because this member is never called because we never construct the object because we never go in that branch of the parser that creates it. I still didnt find how to reach it, but I suspect the more code we'll add to tests the closer we'll get there.